pkg/cli/stress_test.go

```diff
@@ -182,42 +186,45 @@ func worker(id int, zotPort, rootDir string) {
 	}
 }
 
-func setMaxOpenFilesLimit(limit uint64) error {
+func setMaxOpenFilesLimit(limit uint64) (uint64, error) {
 	var rLimit syscall.Rlimit
 
 	err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rLimit)
 	if err != nil {
-		return err
+		return 0, err
 	}
 
 	fmt.Println("Current max. open files ", rLimit.Cur)
+	initialLimit := rLimit.Cur
 	rLimit.Cur = limit
 	fmt.Println("Changing max. open files to ", limit)
 
 	err = syscall.Setrlimit(syscall.RLIMIT_NOFILE, &rLimit)
 	if err != nil {
-		return err
+		return initialLimit, err
 	}
 
 	err = syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rLimit)
 	if err != nil {
-		return err
+		return initialLimit, err
 	}
 
 	fmt.Println("Max. open files is set to", rLimit.Cur)
 
-	return nil
+	return initialLimit, nil
 }
 
 func startServer(c *api.Controller) {
 	// this blocks
 	ctx := context.Background()
 	if err := c.Run(ctx); err != nil {
-		return
+		fmt.Printf("\nerror on starting zot server: %s\n", err)
```

Comment:
could you maybe move `\n` to the previous string/s

Reply:
What previous message? Can you guarantee a specific one?
printf because that was used for printing errors in the rest of the file.